src/cost_estimates/tests/cost_estimators.rs

```diff
@@ -55,6 +60,14 @@ impl CostMetric for TestCostMetric {
     }
 }
 
+const BLOCK_LIMIT_MAINNET_STACKS20: ExecutionCost = ExecutionCost {
```

Comment:
Why is this duplicated here?

Reply:
This has been resolved at least by commit 340985579